packages/fxa-payments-server/server/config/index.js

```diff
@@ -9,12 +9,28 @@ const path = require('path');
 const convict = require('convict');
 
 const conf = convict({
+  amplitude: {
+    disabled: {
```

Comment:
I'm a bit surprised that the content server uses `disabled` already. We generally use `enabled` within config rather than `disabled` because negating negatives is hard - `disabled: false` just takes too much thought.

Reply:
:+1: I'll flip the value here and file a bug to invert the value on the content server, too.